packages/react-aria-components/stories/button-pending.css

```diff
@@ -0,0 +1,27 @@
+.button {
+  position: relative;
+  height: 30px;
+  display: flex;
+  align-items: center;
+  box-sizing: border-box;
+  outline: none;
+}
```

Comment:
A focus state would be nice, to know when it's focused when testing with keyboard.